bless/request/bless_request_host.py

```diff
@@ -29,13 +29,13 @@ class BlessHostSchema(Schema):
     public_key_to_sign = fields.Str(validate=validate_ssh_public_key, required=True)
 
     @validates_schema(pass_original=True)
-    def check_unknown_fields(self, data, original_data):
+    def check_unknown_fields(self, data, original_data, **kwargs):
```

Comment:
how come we need to add `kwargs`  to all these functions?

Reply:
it is the change from the Marshmallow package

Reply:
I have tested the change following the instruction: 
https://github.com/lyft/bless-private/blob/master/docs/runbook.md